hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/traversers/Vertices.java

```diff
@@ -51,6 +52,10 @@ public Iterator<Vertex> vertices(HugeGraph g) {
                           this.label == null), "No source vertices provided");
         Iterator<Vertex> iterator;
         if (this.ids != null && !this.ids.isEmpty()) {
+            E.checkArgument(this.label == null,
+                            "Just provide one of ids or label of source vertices");
+            E.checkArgument(props == null || props.isEmpty(),
+                            "Just provide one of ids or properties of source vertices");
```

Comment:
NOTE: it's a `BREAKING CHANGE`, we need change the PR title to `BREAKING CHANGE:  xxx` in the future

related CI error when upgrade the server: https://github.com/apache/incubator-hugegraph-ai/actions/runs/8673001982/job/23783894307